src/cases/components/case-hearings-list/case-hearings-list.component.html

```diff
@@ -31,9 +31,9 @@
              (click)="viewAndEdit(hearing.hearingID)">View or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;
         </ng-container>
         <ng-container *ngIf="isAwaitingActual(hearing.exuiDisplayStatus)">
-          <a id="link-add-or-edit-{{hearing.hearingID}}"
+          <a id="link-view-details-awaiting-{{hearing.hearingID}}"
              href="javascript:void(0)"
-             (click)="addAndEdit(hearing.hearingID)">Add or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+             (click)="viewDetails(hearing)">View details</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
this is some breaking changes, the addAndEdit(hearing.hearingID) shouldn't be changed, addAndEdit is for role caseWorker, while for judicial user it will be viewDetails(hearing)